sklearn/calibration.py

```diff
@@ -546,6 +547,13 @@ def calibration_curve(y_true, y_prob, normalize=False, n_bins=5):
         points (i.e. without corresponding values in y_prob) will not be
         returned, thus there may be fewer than n_bins in the return value.
 
+    strategy : {'uniform', 'quantile'}, (default='uniform')
+        Strategy used to define the widths of the bins.
+        uniform
```

Comment:
put a blank line before this for correct rendering in HTML

Reply:
You're saying a blank line between the lines starting with "strategy" and the line with "uniform"?